grype/presenter/sarif/presenter.go

```diff
@@ -122,6 +123,7 @@ func (pres *Presenter) sarifRules() (out []*sarif.ReportingDescriptor) {
 					// For GitHub reportingDescriptor object:
 					// https://docs.github.com/en/code-security/code-scanning/integrating-with-code-scanning/sarif-support-for-code-scanning#reportingdescriptor-object
 					"security-severity": pres.securitySeverityValue(m),
+					"purls":             [...]string{deriveBomRef(m.Package)},
```

Comment:
Why is this using this specific function instead of just referencing the `m.Package.PURL`? This should not be required to match the `bom-ref`, it should probably just be the value for the PURL

Reply:
Thanks a lot for the review. Wasn't sure and I wanted to compute it the same way CycloneDx does.
I will change it.